tests/Doctrine/ODM/MongoDB/Tests/Functional/Ticket/GH1418Test.php

```diff
@@ -103,4 +105,7 @@ class GH1418Embedded
 {
     /** @ODM\Field(type="string") */
     public $name;
+
+    /** @ODM\AlsoLoad("alsoLoadMe") **/
```

Comment:
In order for this to work you'll have to map it as a field. Changing the mapping to the following works:
```
/**
 * @ODM\Field(type="int")
 * @ODM\AlsoLoad("alsoLoadMe")
 */
public $alsoLoadValue;
```